### PR TITLE
ISSUE #703 - Adding Creative Commons 3.0 License to project base and readme

### PR DESCRIPTION
**Pull request by thatsjet**
Currently, only the .doc and pdf versions of the ASVS mention the license in use. Adding a LICENSE.md file and mention in the Readme.md to make clear the license in use for the whole project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +30,7 @@ The latest published version is version 4.0.1 (dated 2 March 2019), which can be
 * [OWASP Application Security Verification Standard 4.0.1 (GitHub Tag)](https://github.com/OWASP/ASVS/tree/v4.0.1)
 
 The master branch of this repository will always be the "bleeding edge version" which might have in-progress changes or other edits open. The next release target will be version **4.1**.
+
+# License
```
Comment (kingthorin) [inline README.md]: Level 2?

**Comment by kingthorin** [comment]
For reference: CC-BY-SA 3.0 per: https://github.com/OWASP/ASVS/blob/master/4.0/en/0x01-Frontispiece.md#copyright-and-license

**Comment by tghosth** [comment]
If you can fix the codacy issues/conflicts I will merge this. Thanks :)

**Comment by thatsjet** [comment]
Done, checks passed.

**Comment by tghosth** [comment]
Ok, I'll have to take a look when I am near my pc. The following is just a note for myself:
[Creative Commons License](https://creativecommons.org/licenses/by-sa/4.0/ "CC BY-SA 4.0")
[OWASP Flagship](https://www.owasp.org/index.php/Category:OWASP_Project#tab=Project_Inventory)

**Comment by kingthorin** [comment]
It wasn’t a real (git) conflict, it was a codacy conflict.

**Pull request merged**